### ISSUE TITLE
[XMPP] group subjects are annoying

### ISSUE THREAD

**Issue by ghost** [loqui/im#526]
If i have some groups add to a XMPP account, everytime Loqui reconnects the subject message of the group is shown "user has set the subject to: subject description". These messages are repeated again and again. Is it possible to show these messages only once? When i access a group for the first time or the subject was changed.

**Comment by ghost** [comment]
+1

**Comment by ghost** [comment]
If someone else is still interested in that feature open a new issue or pull request.
